### ISSUE TITLE
'window.event' not working properly

### ISSUE THREAD

**Issue by asrdri** [violentmonkey/violentmonkey#1223]
Few months ago this worked well. Example:

```
// ==UserScript==
// @name        New script - youtube.com
// @namespace   Violentmonkey Scripts
// @match       https://www.youtube.com/*
// @grant       none
// @version     1.0
// @author      -
// @run-at      document-start
// @grant       GM_xmlhttpRequest
// ==/UserScript==

setTimeout(assign,2000);

function assign() {
  document.getElementsByClassName('ytd-notification-topbar-button-renderer')[0].addEventListener("click", function test() {
    if (window.event.ctrlKey === true) {
      alert("CTRL pressed");
    }  else if (window.event.shiftKey === true) {
      alert("SHIFT pressed");
    } else {
      alert("Nothing pressed");
    }
  });
}
```
Go to https://www.youtube.com/c/google/videos and click the Notifications button at the top right, while Shift or Ctrl is pressed. Then try without them pressed. You'll see incorrect outputs.
Tested in Chrome and Firefox.

With some experimentation I found that it would work fine if I set this:
```
// @grant       none
```
But it will break again if I set it like this:
```
// @grant       none
// @grant       none
```

**Comment by tophf** [comment]
This will be fixed in the next release, thanks for reporting.
Meanwhile use the actual event parameter instead of the deprecated `window.event`, more on [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Window/event).

As for multiple `@grant none` behaving differently, that's also a bug, fixed in 52a738e8.

**Comment by tophf** [comment]
`event` is the same as `window.event` if you didn't declare it as a function parameter:

```js
function(event) {
  //////
}
```
